subprojects/tooling-api/src/crossVersionTest/groovy/org/gradle/integtests/tooling/m8/ToolingApiLoggingCrossVersionSpec.groovy

```diff
@@ -29,12 +29,6 @@ class ToolingApiLoggingCrossVersionSpec extends ToolingApiLoggingSpecification {
 
     def setup() {
```

Comment:
I _think_ tooling API tests won't be affected by what you did in `GradleExecuter` But that's not a problem, because when we use the tooling API we never print the welcome message anyway, right (because `CommandLineActionFactory` is not used there)?

Reply:
Correct. This test case which verifies if the Tooling API renders the message does not pass yet: https://github.com/gradle/gradle/blob/master/subprojects/launcher/src/integTest/groovy/org/gradle/launcher/cli/NotificationsIntegrationTest.groovy#L68